src/time.cr

```diff
@@ -496,60 +829,71 @@ struct Time
     Format.new(format).format(self)
   end
 
-  # Formats this time using the given format (see `Time::Format`)
-  # into the given *io*.
+  # Formats this `Time` according to the pattern in *format* to the given *io*.
+  #
+  # See `Time::Format for details.
   def to_s(format : String, io : IO) : Nil
     Format.new(format).format(self, io)
   end
 
-  # Parses a Time in the given *time* string, using the given *pattern* (see
-  # `Time::Format`).
+  # Parses a `Time` from *time* string using the given *pattern*.
+  #
+  # See `Time::Format` for details.
   #
   # ```
   # Time.parse("2016-04-05", "%F") # => 2016-04-05 00:00:00 +01:00
   # ```
-  def self.parse(time : String, pattern : String, location = nil) : Time
+  def self.parse(time : String, pattern : String, location : Location? = nil) : Time
     Format.new(pattern, location).parse(time)
   end
 
-  # Returns the number of seconds since the Epoch for this time.
+  # Returns the number of seconds since Unix epoch (`1970-01-01 00:00:00 UTC`).
```

Comment:
the unix epoch and ditto throughout.

Reply:
I'm pretty sure I had already address this, but it's typically spelled `Unix epoch`, `UNIX epoch` or `Unix Epoch` and I don't think there is a strong point for any of them over the others.